test/helpers/helpers.go

```diff
@@ -108,6 +135,27 @@ func CreateRuntimeObject(f *test.Framework, ctx *test.TestCtx, obj runtime.Objec
 		})
 }
 
+// CreateRuntimeObject creates a runtime object using the test framework.
+func DeleteRuntimeObject(f *test.Framework, ctx *test.TestCtx, obj runtime.Object) error {
+	err := f.Client.Delete(
+		context.TODO(),
+		obj,
+	)
+
+	return err
+}
+
+// CreateRuntimeObject creates a runtime object using the test framework.
```

Comment:
I see that an helper exists for this at https://github.com/operator-framework/operator-marketplace/blob/master./test/helpers/helpers.go#L100. Can't that be reused?